src/lib/bundler/getWebpackConfig.test.ts

```diff
@@ -1,7 +1,11 @@
 import { getWebpackConfig } from './getWebpackConfig'
 
 describe('getWebpackConfig()', () => {
-  it('should generate webpack config', () => {
-    getWebpackConfig({ pages: ['foo.js'], plugins: [] })
+  it('should generate devserver webpack config', () => {
+    getWebpackConfig({ pages: ['foo.js'], plugins: [], devServer: true })
```

Comment:
Sorry - might be being thick - do these tests actually check the resulting webpack config to confirm it is the expected one?

Reply:
They don't. It basically just confirms that it doesn't throw.Really testing that the config is correct would probably require end-end tests (eg. selenium tests against a running server) which we don't currently have but probably should.